commands/permissions.go

```diff
@@ -4,35 +4,44 @@
 package commands
 
 import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
 	"github.com/mattermost/mattermost-server/v5/model"
 
 	"github.com/mattermost/mmctl/client"
-	"github.com/mattermost/mmctl/printer"
 
 	"github.com/spf13/cobra"
 )
 
+const flagAncillaryPermissions = "ancillary"
+
 var PermissionsCmd = &cobra.Command{
 	Use:   "permissions",
 	Short: "Management of permissions and roles",
 }
 
 var AddPermissionsCmd = &cobra.Command{
-	Use:     "add [role] [permission...]",
-	Short:   "Add permissions to a role (EE Only)",
-	Long:    `Add one or more permissions to an existing role (Only works in Enterprise Edition).`,
-	Example: `  permissions add system_user list_open_teams`,
-	Args:    cobra.MinimumNArgs(2),
-	RunE:    withClient(addPermissionsCmdF),
+	Use:   "add [role] [permission...]",
+	Short: "Add permissions to a role (EE Only)",
```

Comment:
```suggestion

Reply:
Apologies, it looks like some of the comments didn't get post correctly, fixing...